Reflection/ReflectionType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * The ReflectionType class reports information about a function's parameters.
+ *
+ * @link https://www.php.net/manual/en/class.reflectiontype.php
+ * @since 7.0
+ */
+abstract class ReflectionType implements Stringable
+{
+    /**
+     * Checks if null is allowed
+     *
+     * @link https://php.net/manual/en/reflectiontype.allowsnull.php
+     * @return bool Returns {@see true} if {@see null} is allowed, otherwise {@see false}
+     * @since 7.0
+     */
+    public function allowsNull(): bool
+    {
+    }
+
+    /**
+     * Checks if it is a built-in type
+     *
+     * @link https://php.net/manual/en/reflectiontype.isbuiltin.php
+     * @return bool Returns {@see true} if it's a built-in type, otherwise {@see false}
+     * @since 7.0
+     *
+     * @deprecated Since 8.0 this method has been removed from the ReflectionType
```

Comment:
Could you please remove `Since` word before version
`@deprecated 8.0`
Otherwise deprecation will not be highlighted properly by PhpStorm due to wrong parsing of this comment